package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "pokedex",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "storybook-generate": "sb-rn-get-stories",
+    "storybook-watch": "sb-rn-watcher"
+  },
+  "dependencies": {
+    "expo": "~49.0.5",
+    "expo-status-bar": "~1.6.0",
+    "react": "18.2.0",
+    "react-native": "0.72.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.20.0",
+    "@react-native-async-storage/async-storage": "^1.19.1",
+    "@react-native-community/datetimepicker": "^7.4.1",
+    "@react-native-community/slider": "^4.4.2",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-controls": "^6.5.16",
+    "@storybook/addon-ondevice-actions": "^6.5.4",
+    "@storybook/addon-ondevice-controls": "^6.5.4",
+    "@storybook/react-native": "^6.5.4",
+    "@types/react": "~18.0.14",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
+    "@typescript-eslint/parser": "^6.2.0",
+    "babel-loader": "^8.3.0",
+    "eslint": "^8.45.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-react": "^7.33.0",
+    "jest": "^29.6.1",
```

Comment:
Is that possible to use react testing library?

Reply:
in the react native documentation, they suggested to use Jest, i dont really know if we can use react-testing-library yet, but i sure can search for it

Reply:
Yeah, we'll stay using jest, but we're going to have the support of the testing library for that as well. Jest it's the framework and the Testing library it's a package to help you test components.

https://testing-library.com/docs/react-native-testing-library/intro/